src/components/RuleCard/index.jsx

```diff
@@ -645,11 +641,14 @@ function RuleCard({
                 <List>
                   <ListItem className={classes.listItem}>
                     <ListItemText
-                      className={classes.comment}
+                      title={rule.comment}
                       primaryTypographyProps={{
                         component: 'div',
                         className: classes.primaryText,
                       }}
+                      secondaryTypographyProps={{
+                        className: classes.textEllipsis,
+                      }}
```

Comment:
Most cards (at least in development) have single line comments. Optimizing for a single line would reduce the spacing between cards since we gave 3 lines of height to each card which made the spacing between cards without comments really big. This patch will show `...` when a comment is > 1 line. Hovering on the comment should show the full comment.

Reply:
I think a tooltip is probably enough. We can look into a better solution later if it turns out not to be.